THIRD-PARTY-NOTICE.txt

```diff
@@ -0,0 +1,7765 @@
+Azure SDK for Java uses third-party libraries or other resources that may be
+distributed under licenses different than the Azure SDK for Java software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           azjavasdkhelp@microsoft.com
+
+The attached notices are provided for information only.
+
+License notice for Azure SDK for Java
+-------------------------------
+
+Copyright (c) 2015 Microsoft. All rights reserved.
```

Comment:
Is this the right text? All our class files use the following license (no 2015 in the license we use for track 2). Maybe we should update LICENSE.txt too?
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```